.env.example

```diff
@@ -1,3 +1,6 @@
 # Create your own bot via https://discord.com/developers/applications
 DISCORD_TOKEN=XXXXXX
-DISCORD_APPID=XXXXXX
\ No newline at end of file
+DISCORD_APPID=XXXXXX
+
+RSPOTIFY_CLIENT_ID=XXXXXX
+RSPOTIFY_CLIENT_SECRET=XXXXXX
```

Comment:
This is a `SPOTIFY_*` I would assume? The `rspotify` library just uses them to authenticate with Spotify.

Reply:
[`Credentials::from_env`](https://docs.rs/rspotify/0.11.4/rspotify/struct.Credentials.html#method.from_env) looks for `RSPOTIFY_*` in env.
Alternatively, we can write our own variables, manually fetch from our env file and then use:

```rust
let creds = Credentials::new(SPOTIFY_CLIENT_ID, SPOTIFY_CLIENT_SECRET);
```

Reply:
Yes <3